spec/schema.json

```diff
@@ -53,6 +53,12 @@
             "schema": {
               "$ref": "#/definitions/ResourceCreated"
             }
+          },
+          "400": {
+            "description": "Company was ineligible for objecting to strike off",
```

Comment:
Minor, but suggest changing the 'was' to 'is' in the description. It's at the point the end-point is called that the company is ineligible - we're not doing an historical check.

Reply:
Changed this to "is"